packages/SwingSet/src/kernel/kernel.js

```diff
@@ -645,7 +696,9 @@ export default function buildKernel(
     insistCapData(vatParameters);
 
     // eslint-disable-next-line no-use-before-define
-    assert(vatWarehouse.lookup(vatID));
+    if (!vatWarehouse.lookup(vatID)) {
```

Comment:
Note to self: need a test for this, bootstrap vat should use the admin facet to terminate a vat and then send an update, make sure vat-admin vat/device intermediate messages queue it in the useful order, then make sure nothing explodes